app/src/main/res/xml/provider_paths.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<paths xmlns:android="http://schemas.android.com/apk/res/android">
+    <external-path
+        name="Download"
+        path="Download/" />
```

Comment:
Is it possible to define some kind of wildcard path here?

- Currently the app crashes when saving an image (long press image -> Save image) because it's not in the download path
- There might be devices where the download directory is named differently. In code we are using the Environment.DIRECTORY_* constants.